torch/nested/_internal/ops.py

```diff
@@ -204,6 +204,7 @@ def extract_kwargs(arg):
     kwargs = {
         "offsets": arg.offsets(),
         "_metadata_cache": arg._metadata_cache,
+        "_ragged_idx": arg._ragged_idx,
```

Comment:
Note for posterity: this was left out here previously as `_ragged_idx != 1` was assumed to imply non-contiguity, and we generally don't preserve non-contiguity through out-of-place op invocations.